dockerfile/cuda11.1.1.dockerfile

```diff
@@ -85,6 +89,19 @@ RUN cd /tmp && \
     cd /tmp && \
     rm -rf nccl
 
+# Install Docker and NVIDIA Container Toolkit
+RUN curl -fsSL https://download.docker.com/linux/ubuntu/gpg | \
+    gpg --dearmor -o /usr/share/keyrings/docker-archive-keyring.gpg && \
+    echo \
+    "deb [arch=amd64 signed-by=/usr/share/keyrings/docker-archive-keyring.gpg] https://download.docker.com/linux/ubuntu \
+    $(lsb_release -cs) stable" | tee /etc/apt/sources.list.d/docker.list > /dev/null && \
+    distribution=$(. /etc/os-release;echo $ID$VERSION_ID) && \
+    curl -s -L https://nvidia.github.io/nvidia-docker/gpgkey | APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=1 apt-key add - && \
+    curl -s -L https://nvidia.github.io/nvidia-docker/$distribution/nvidia-docker.list | \
+    tee /etc/apt/sources.list.d/nvidia-docker.list && \
+    apt-get update && \
+    apt-get install -y --no-install-recommends docker-ce docker-ce-cli containerd.io nvidia-docker2
```

Comment:
do not need nvidia container toolkit, it's for docker daemon, to use docker-in-docker, only client part is needed

Reply:
Have followed https://github.com/docker-library/docker/blob/f30c3dd6f0665b2b34e520f9e9348dcaa8755de9/20.10/Dockerfile#L32